### PR TITLE
[REVIEW] doc and API fixes

### PR DESCRIPTION
**Pull request by BradReesWork**
Updated docs to fix missing arguments
Added NetworkX like API

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Review by rlratzel — APPROVED** [review]
LGTM. I had one minor suggestion/question which need not hold up my approval.

**Comment by BradReesWork** [comment]
rerun tests

**Review by Iroy30 — APPROVED** [review]
LGTM

File: python/cugraph/structure/graph.py
Comment (afender) [inline python/cugraph/structure/graph.py:987]:
```suggestion
            GPU DataFrame of size V (the default) or the size of the given
```

**Review by afender — APPROVED** [review]
lgtm

**Pull request merged**
